server/README.md

```diff
@@ -27,6 +27,7 @@ pre-requiste: node.js is already installed on your machine
                 }
         }
 ```
+* POST - https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/insertFormData 
```

Comment:
Could you just mention somewhere here that it returns profile_id?

Reply:
Maybe a sample response/request JSON - look at the readme on main as an example